src/scss/components/_badges.scss

```diff
@@ -18,25 +19,37 @@ $badge-radius       : 14px !default;
 
 @mixin badge() {
 
-    .badge {
+    .c-badge {
         background-color: $badge-bgColor;
         padding: $badge-padding;
     }
 
-        .badge--rounded {
+        .c-badge--rounded {
```

Comment:
things seem indented by one tab below here

Reply:
I thought that was the convention for modifiers etc? I was [following the style used in the `cards` component](https://github.com/justeat/fozzie/blob/master/src/scss/components/_cards.scss).

Reply:
i think if it was nested like so it would make sense:

```scss
.c-card {
  &--noPad {
    ...
  }
  &--noPadVertical {
    ...
  }
}
```

but as it all sits inside a single level i would keep them tabbed in line

Reply:
That's the current convention we use across other components.  Prob best to leave it like this and then if we want to change it do so all in one go (can discuss if we think it's wrong atm).